layers/+lang/javascript/packages.el

```diff
@@ -86,6 +86,9 @@
                      (cons 'javascript-backend value))))
     :config
     (progn
+      ;; file extensions
+      (push '("\\.js\\'"  . js2-mode) auto-mode-alist)
+      (push '("\\.jsx\\'" . js2-mode) auto-mode-alist)
```

Comment:
I would prefer using `add-to-list` instead of push here to prevent stacking this on top of the list with each `SPC f e R`. Also I would put these in `:init` instead of  `:config`, also check use-package's `:mode` (example here: https://github.com/syl20bnr/spacemacs/blob/develop/layers/+readers/pdf/packages.el#L18).

Reply:
A better, an even better and an even-even-better solution 😸